### ISSUE TITLE
Accessibility: Auto-playing audio

### ISSUE THREAD

**Issue by jason-me** [stakwork/sphinx-tribes-frontend#1051]
## Issue
The video player has audio and video content which automatically plays for more than 5 seconds upon landing on the page. This content can be disorienting to screen reader users who navigate the video player controls audibly.

## Expected Behavior
- [WCAG 2.2 A 1.4.2 Audio Control](https://www.w3.org/TR/WCAG22/#audio-control) requires that if any audio on a web page plays automatically for more than 3 seconds, either a mechanism is available to pause or stop the audio, or a mechanism is available to control audio volume independently from the overall system volume level.
- Technically this standard is met because the video player has an accessible pause button, however it may still be disorienting to screen reader users who cannot find the control quickly
- As a best practice screen reader users expect to be able to navigate to a page without audio automatically playing over their screen reader
## Acceptance Criteria
- [x] Research the video player and determine if there is a method to pause the video on page load or at least mute the volume.
- [x] Test with a screen reader JAWS, VoiceOver, NVDA, Orca, Talkback, etc. 
- [x] Ensure screen reader users can navigate the page without the interruption pf automatically playing video.

_Linked to tracking issue: #456_

**Comment by jason-me** [comment]
Closed by https://github.com/stakwork/sphinx-tribes-frontend/pull/1054
